Glossary add label (#1359)
zkSNACKs/WasabiDoc: docs/glossary/Glossary-PrivacyWasabi.md

Read more: [What is the dust threshold](/FAQ/FAQ-UseWasabi.html#what-is-the-dust-threshold)
:::

:::details
### Label

> A label can be added to a coin, like a small note, so it is easy to recognize later on who knows about this coin.
> Good labelling can help the user and the wallet to make better privacy consious decisions.

Reviewer: ```suggestion
A label can be added to a coin, as a small note on who knows this coins belongs to you.
Good labelling can help the user and the wallet to make better privacy conscious decisions later on when spending.
```
